src/coap_address.c

```diff
@@ -643,7 +696,7 @@ coap_resolve_address_info(const coap_str_const_t *address,
   error = getaddrinfo(addrstr, NULL, &hints, &res);
 
   if (error != 0) {
-    coap_log_warn("getaddrinfo: %s: %s\n", addrstr, gai_strerror(error));
+    coap_log_warn("getaddrinfo: %s: %d\n", addrstr, error);
```

Comment:
My preference would be to still have `gai_strerror()` to translate the `getaddrinfo()` that will have more meaning to most developers.  It is also used elsewhere in the code.

I agree that some platforms do not support `gai_strerror()` and stub functions have had to be written.